### DISCUSSION TITLE
Welcome to puppeteer-on-termux Discussions!

### DISCUSSION THREAD

**Discussion by rishabhrpg** [rishabhrpg/puppeteer-on-termux#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by md-fahad-ali32** [comment]
Hello sir i am installing alpine but i can't install nodejs apk add nodejs shows erorr also chromium

**Comment by tdee912** [comment]
![WhatsApp Image 2023-02-26 at 3 19 34 AM](https://user-images.githubusercontent.com/9406349/221377932-d28b8282-bca4-40a4-b567-b1ddf2f8036e.jpeg)
Hello brother, after succes run index.js, i can't install whatsapp-web.js on alpine/puppeteer-on-termux directory. 
any solution?
thanks
